font/src/darwin/mod.rs

```diff
@@ -220,6 +260,22 @@ impl Rasterizer {
             },
         }
     }
+
+    // helper to try load a font for a specific glyph
+    fn attempt_load_font(
```

Comment:
This should probably be called `maybe_get_glyph` or at least something related to glyphs instead of fonts.